lib/components/app/desktop-nav.tsx

```diff
@@ -9,6 +9,7 @@ import { accountLinks, getAuth0Config } from '../../util/auth'
 import { AppConfig } from '../../util/config-types'
 import { AppReduxState } from '../../util/state-types'
 import { DEFAULT_APP_TITLE } from '../../util/constants'
+import { isMobile } from '@opentripplanner/core-utils/lib/ui'
```

Comment:
Install the ESLint extension for your IDE and address the warning about import order.